### ISSUE TITLE
[17] Informative text is embedded in image

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#290]
This issue was identified as part of the accessibility testing performed on the DTA website Jan 2017: https://drive.google.com/open?id=0B3SXZ9dr83v0Y2cwV21HdUZ1STQ

Applies: Image on How we're tracking - progress in our Digital Delivery Hubs
Type: WCAG failure – 1.4.5 Images of Text
Platform: Desktop and mobile
Severity: Medium

The progress for Delivery Hubs has text that is embedded in an image.

![image](https://cloud.githubusercontent.com/assets/2324569/23888440/0a664d60-08dc-11e7-80d5-5377d95c6c78.png)

Although adding alt text will satisfy 1.1.1 Non-Text Content and provides an alternative for screen readers, text embedded in images still fails 1.4.5 Images of Text.
Text embedded in images, also known as images of text, cannot be adjusted by people with low vision or others who may need to change the font type, size or colour of the text.

**Recommendation**

Use true text styled using CSS.

Alternatively, add a text description underneath the image.

**Comment by bjgordon** [comment]
Will either change the image to a table, or add a link to a new page with "Read full text of image".

@AndrewArch  - suggests adding a border shadow so it's obvious it's an image and not content.
